src/styles/styles.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React, { createContext, useContext, useMemo } from 'react';
+
+import * as themes from './themes';
+import base from './base';
+
+const DEFAULT_THEME_NAME = 'chromeLight';
+
+const ThemeContext = createContext(base(themes[DEFAULT_THEME_NAME]));
+
+/**
+ * Hook to get the component styles for the current theme.
+ */
+export const useStyles = key => {
```

Comment:
rename to baseStylesKey and add a little comment/jsdoc? Might help clarify the usage.